client/src/Components/Overview/ProductOverview.jsx

```diff
@@ -1,12 +1,12 @@
 import React, { useContext } from 'react';
-import currentProducts from '../../Contexts/CurProdContext.js';
+import currentProducts from '../../Contexts/CurProdContext';
```

Comment:
Out of curiosity, what did removing the .js change? What error/indication did you receive to make that change?